VERSIONING.md

```diff
@@ -0,0 +1,63 @@
+# Versioning strategy
```

Comment:
The goal of the metapackage is to provide you with a bundle of PyAnsys libraries that are known to be compatible with the specific version of your Ansys installation. That's why you get so "many" packages installed. It is also a nice way of providing a unified, compatible environment between all libraries since they are known not to have clashing dependencies.

We could open the door to defining specific targets though it might be a bit of an overkill IMO... 

Anyway, if users want to install ``ansys-grantami-serverapi-openapi`` only, then what they should do is install that package alone. If your package is compatible only with one specific Ansys install, then you should do some checks internally or announce the compatibility tables in the documentation.

In any case, as @jorgepiloto said, I encourage you all to follow the semantic versioning scheme that all PyAnsys libraries are using. The metapackage is an exception since it is dedicated audience is the unified install, as well as users that want to ensure themselves that the given versions are compatible with their existing install, and not caring about latest features... (which shouldn't be the case, everybody wants to have the latest and greatest if possible)...

Reply:
See https://didactic-system-49758fff.pages.github.io/docs/adrs/0012-python-api-versioning/